src/makeRobotAccountant.js

```diff
@@ -4,9 +4,19 @@
  *
  * @return {function}
  */
-
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 0;
+
+  return function(value1) {
+    count++;
 
+    return function(value2) {
+      if (count % 2 !== 0 || count <= 3) {
+        return value1 + value2;
+      }
+
+      return 'Bzzz... Error!';
```

Comment:
Advice: better start with the error conditions.
```js
if (!passCondition) {
  return 'Not good!';
}

// do work
```

Reply:
fixed , I hope I understood you correctly=))